build-system/npm-publish/write-package-files.js

```diff
@@ -60,6 +60,7 @@ async function writePackageJson() {
         import: './dist/component-react.module.js',
         require: './dist/component-react.js',
       },
+      './styles.css': './dist/styles.css',
```

Comment:
I was wondering about that too but didn't see a way to easily get that information. It would be nice to only add this if there is a stylesheet. Not sure of the implications if the file doesn't exist. Perhaps worst case your IDE just shows you an incorrect autocompletion?

Reply:
This sounds like something that should be figured out before we merge this PR. How about manually generating test packages with/without a style file and seeing what they do?

Reply:
When does this step occur? If it is after the `dist` folder was already created, then you could literally do a check for the file's existence via `fs.existsSync` (or ideally a promise-based API if you can find a suitable one)